### ISSUE TITLE
Composer broken installation instruction

### ISSUE THREAD

**Issue by pszalko** [tpay-com/omnipay-tpay#1]
It seems that composer installation instruction is broken in README.
After setting up project as stated in README, composer says it can't install `omnipay/tpay` lib.

My `composer.json` file:
```
{
    "name": "tpay-playground",
    "type": "project",
    "require": {
        "omnipay/tpay": "~1.0"
    }
}
```

Composer output:
```
$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package omnipay/tpay could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

**Comment by piotrjozwiak** [comment]
This repository is not yet available at composer, because we still wait for this [pull request](https://github.com/thephpleague/omnipay/pull/477) to be accepted and also for adding omnipay/tpay to packagist by omnipay owners.

**Comment by pszalko** [comment]
Ok, understood, but is this project stable enough to use it in production app? 
Do you plan any further changes that could break its backward compatibility to the current state?

**Comment by piotrjozwiak** [comment]
Of course, it was well tested and we confirm that all provided methods work well.
We do not plan to make any structural changes or to break backward compatibility so please feel free to use it. :)
Please keep in mind this repository requires PHP 5.6+

**Comment by pszalko** [comment]
Great! Thanks.

Maybe it's worth noticing in README for less experienced composer users that they can install this package via `repository` channel until it's not accepted by packagist. 

Anyway, thank you for clearing out this issue.

**Comment by czerwonyd** [comment]
I know it's been over one and a half years since you last spoke but maybe it would be a good idea to create your own package on packagist? Omnipays don't seem to be bothered about the comment in your PR.

**Comment by pszalko** [comment]
+1 for creating your own package on packagist. It's been 2 years and I still have a reference to tpay `dev-master` added to our composer.json file.

**Comment by pszalko** [comment]
I have created my own fork and published it on composer: https://packagist.org/packages/mobtitude/omnipay-tpay for both omnipay v2 and v3.

@piotrjozwiak Please check composer.json in my repo to see what changed. I think it would be great if official tpay repo was available at composer and we could finally close this issue :)
